src/components/CategoryOverview/CategoryOverview.scss

```diff
@@ -155,7 +155,7 @@
     flex: none;
     width: 100%;
     margin-right: 0;
-    margin-top: 8px;
+    margin-top: 16px;
```

Comment:
Unrelated: gap between the 'captains pick' pair of cards on the city overview page.